pkg/node/discovery/discovery_test.go

```diff
@@ -0,0 +1,47 @@
+// This file is part of MinIO Direct CSI
```

Comment:
we can remove this file as it is not used

Reply:
Done